### PR TITLE
Feature/connections

### PR DESCRIPTION
**Pull request by iykekings**
# Description

Describe change or new feature here.

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by iykekings** [comment]
Merge but don't delete branch 🔥

File: api/controllers/connection.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        .filter(c => c.sender_user_id === parseInt(userId, 10))
+        .map(m => {
+          return {
+            id: m.id,
+            accepted: m.accepted,
+            pending: m.pending,
+            connection: m.request_user
+          };
+        });
+      const recieved = connections
```
Comment (ruihildt) [inline api/controllers/connection.js]: typo: `received`

Comment (ruihildt) [inline api/controllers/connection.js]: Typo `received`

Comment (iykekings) [reply api/controllers/connection.js]: I will fix that.... But it has to be mergers now so Katy can work with it on the front end.... I'm away from my PC. Also don't delete this branch after merging

**Pull request merged**